app/javascript/components/common/CommunitySolution.tsx

```diff
@@ -19,23 +19,43 @@ const PublishDetails = ({ solution }: { solution: CommunitySolutionProps }) => {
         solution.publishedAt
       )}`}</time>
       <div className="--counts">
+        {solution.representationNumPublishedSolutions ? (
```

Comment:
Just to be clear: this is `false` if `representationNumPublishedSolutions == 0`, but that's probably what we want, right?

Reply:
Yes. Thanks for checking.